lib/Bindings/Tcl/circt_tcl.cpp

```diff
@@ -0,0 +1,110 @@
+#include <stdlib.h>
+#include <tcl.h>
+
+#include "Circt.h"
+#include "circt/Dialect/Comb/CombDialect.h"
+#include "circt/Dialect/FIRRTL/FIRRTLDialect.h"
+#include "circt/Dialect/HW/HWDialect.h"
+#include "circt/Dialect/SV/SVDialect.h"
+#include "mlir/CAPI/IR.h"
+#include "mlir/Parser.h"
+#include "mlir/Support/FileUtilities.h"
+#include "llvm/Support/SourceMgr.h"
+
+int returnErrorStr(Tcl_Interp *interp, const char *error) {
```

Comment:
It might be convenient to take a Twine so errors can be built up more easily.

Reply:
What do you mean?

Reply:
Twine is an LLVM class that makes it easy and efficient to accept concatenated strings in APIs: https://llvm.org/docs/ProgrammersManual.html#the-twine-class